tests/foreman/api/test_syncplan.py

```diff
@@ -4,11 +4,16 @@
 from fauxfactory import gen_string
 from robottelo import entities
 from robottelo.common.decorators import (
-    data, run_only_on, stubbed)
+    data,
+    run_only_on,
+    skip_if_bug_open,
+    stubbed,
+)
 from robottelo.test import APITestCase
 # (too-many-public-methods) pylint:disable=R0904
 
 
+@stubbed
```

Comment:
Try running with py.test or plain unittest. Nose masks error when using `@stubbed`. Also the idea of the stubbed is to skip tests and as you can see when used in the wrong way it removes tests from results even thought they had to be counted.